Makefile

```diff
@@ -112,3 +112,6 @@ check-dependencies:
 	which hg >/dev/null 2>&1 || apt update && apt install -y mercurial
 	$$GOPATH/bin/dep version || (export DEP_RELEASE_TAG=v0.5.0; curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh)
 	dep status
+
+fmt:
+	hack/verify-gofmt.sh
```

Comment:
@jichenjc Can you please move that into the existing `lint` Maketarget?

Reply:
How about renaming the `lint` target to `verify`?

Reply:
SGTM